src/ctap/storage.rs

```diff
@@ -987,7 +1011,7 @@ mod test {
         let mut rng = ThreadRng256 {};
         let mut persistent_store = PersistentStore::new(&mut rng);
 
-        // Master keys stay the same between resets.
+        // Master keys stay the same between calls.
```

Comment:
I guess the English was wrong. We should probably keep the intent by suffixing "within the same CTAP reset cycle" or something like that, no?

Reply:
I'm not an authority for the English language, but I wasn't sure about its meaning, so I changed it. :)
You other formulation is good, so I changed it to that.